src/App.js

```diff
@@ -17,7 +17,30 @@ const styles = theme => ({
     paddingTop: theme.spacing.unit * 20,
   },
 });
-
+class Tile extends React.Component{
```

Comment:
@saini-himanshu : Will need you to move this to file named `Tiles.js` inside a folder named `Tiles` inside `components` folder!